rllib/core/models/tests/test_cnn_transpose_heads.py

```diff
@@ -0,0 +1,93 @@
+import itertools
+import unittest
+
+from ray.rllib.core.models.configs import CNNTransposeHeadConfig
+from ray.rllib.utils.framework import try_import_tf, try_import_torch
+from ray.rllib.utils.test_utils import framework_iterator, ModelChecker
+
+_, tf, _ = try_import_tf()
+torch, _ = try_import_torch()
+
+
+class TestCNNTransposeHeads(unittest.TestCase):
+    def test_cnn_transpose_heads(self):
+        """Tests building DeConv heads properly and checks for correct architecture."""
+
+        # Loop through different combinations of hyperparameters.
+        inputs_dimss = [[1], [50]]
+        initial_image_dims = [4, 4, 96]
+        cnn_transpose_filter_specifierss = [
+            [[48, 4, 2], [24, 4, 2], [3, 4, 2]],  # double image 3x (32) into an RGB
+            [[48, 4, 2], [24, 4, 2], [1, 4, 2]],  # double image 3x (32) into grayscale
+            [[3, 4, 3]],  # triple image (->12x12) into an RGB
+            [[1, 7, 3]],  # triple image (->12x12) w/ larger filter into grayscale
```

Comment:
What do "3x (32)"  and "(->12x12)" mean?
Maybe it's me, but I think these comments could be clearer.

Reply:
Makes sense, will clarify ..

Reply:
fixed